src/App.js

```diff
@@ -12,138 +12,101 @@ import People from './components/People';
+
+    componentDidMount = () => {
+      // loadPeopleFromServer
+      this.loadData(this.props.people_api, "people")
+      // loadNicetiesFromMe
+      this.loadData(this.props.from_me_api, "fromMe")
+      // loadNicetiesForMe
+      this.loadData(this.props.for_me_api, "niceties")
+      // loadSelfInfo
+      this.loadData(this.props.self_api, "selfInfo")
+    }
+
+    handleSelect = (eventKey) => {
         this.setState({currentview: eventKey});
-    },
-    selectComponent: function(idx) {
-        switch(idx) {
-        case "write-niceties":
-            $('.dropdown-toggle').text('Write Niceties');
-            $('.dropdown-toggle').append('<span class="caret"></span>');
-            return <People people={this.state.people}
-                            fromMe={this.state.fromMe}
-                            save_nicety_api={this.props.save_nicety_api} />
-        case "view-niceties":
-            $('.dropdown-toggle').text('Niceties About You');
-            $('.dropdown-toggle').append('<span class="caret"></span>');
-            return <NicetyDisplay niceties={this.state.niceties} />
-        case "admin":
-            $('.dropdown-toggle').text('Admin');
-            $('.dropdown-toggle').append('<span class="caret"></span>');
-            return <Admin admin_edit_api={this.props.admin_edit_api}/>
-        default:
+    }
+
+    selectComponent = (idx) => {
+        switch (idx) {
+            case "write-niceties":
+                $('.dropdown-toggle').text('Write Niceties');
+                $('.dropdown-toggle').append('<span class="caret"></span>');
+                return <People
+                    people={this.state.people}
+                    fromMe={this.state.fromMe}
+                    save_nicety_api={this.props.save_nicety_api}/>
+            case "view-niceties":
+                $('.dropdown-toggle').text('Niceties About You');
+                $('.dropdown-toggle').append('<span class="caret"></span>');
+                return <NicetyDisplay niceties={this.state.niceties}/>
+            case "admin":
+                $('.dropdown-toggle').text('Admin');
+                $('.dropdown-toggle').append('<span class="caret"></span>');
+                return <Admin admin_edit_api={this.props.admin_edit_api}/>
+            default:
         }
-    },
-    render: function() {
+    }
+
+    render() {
         let selectedComponent = this.selectComponent(this.state.currentview);
         let adminMenu = null;
         if (this.state.selfInfo.admin === true) {
             adminMenu = (<MenuItem eventKey="admin">Admin</MenuItem>);
         }
         return (
             <div className="App">
-                <Navbar fixedTop id="main_nav">
-                <div id="title">
-                    recurse<br />
-                    nice-<br />
-                    ties
-                </div>
-                <img id="octotie" src={octotie} height="153"/>
+                <Navbar fixedTop="fixedTop" id="main_nav">
```

Comment:
The [react-bootstrap `<Navbar>` component](https://react-bootstrap-v3.netlify.app/components/navbar/#navbars) exposes [`fixedTop` and its relatives as boolean properties](https://react-bootstrap-v3.netlify.app/components/navbar/#navs-props-navbar). JSX allows for two ways of setting a boolean property to `true`: `<example x />` and `<example x={true} />`. I don't think it's correct to pass a string here.

```suggestion
                <Navbar fixedTop id="main_nav">
```